src/client/Microsoft.Identity.Client/MsalServiceExceptionFactory.cs

```diff
@@ -93,6 +93,26 @@ internal class MsalServiceExceptionFactory
             return ex;
         }
 
+        internal static MsalServiceException FromGeneralHttpResponse(
+          string errorCode,
+          string errorMessage,
+          HttpResponse httpResponse,
+          Exception innerException = null)
+        {
+            MsalServiceException ex = null;
+
+            if (ex == null)
```

Comment:
ex is null

Reply:
:D Thanks for catching that, removed some code and forgot to update that.